docs/table/operations.rst

```diff
@@ -302,6 +303,8 @@ to perform the aggregation::
                 17.0
   15.699999999999998
```

Comment:
Depending on how the conversation about ``+FLOAT_CMP`` resolves, this might be good to change as well.

```suggestion
                15.7
```

Reply:
Done!